### ISSUE TITLE
Add custom browser or cli command

### ISSUE THREAD

**Issue by leancode** [Browsers-software/browsers#228]
**Is your feature request related to a problem? Please describe.**
I use "Web Apps" on Mint to organise all my web apps and they all run in their individual contexts. These are not detected by Browsers unfortunately.
E.g. Microsoft Teams has been set up and the desktop shortcut contains the following:

[Desktop Entry]
Version=1.0
Name=Teams Microsoft
Comment=Web App
Exec=brave-browser --app="https://www.microsoft.com/en-us/microsoft-teams/join-a-meeting" --class=WebApp-TeamsMicrosoft6971 --name=WebApp-TeamsMicrosoft6971 --user-data-dir=/home/dom/.local/share/ice/profiles/TeamsMicrosoft6971
Terminal=false
X-MultipleArgs=false
Type=Application
Icon=web-microsoft
Categories=GTK;WebApps;
MimeType=text/html;text/xml;application/xhtml_xml;
StartupWMClass=WebApp-TeamsMicrosoft6971
StartupNotify=true
X-WebApp-Browser=Brave Browser
X-WebApp-URL=https://www.microsoft.com/en-us/microsoft-teams/join-a-meeting
X-WebApp-CustomParameters=
X-WebApp-Navbar=false
X-WebApp-PrivateWindow=false
X-WebApp-Isolated=true

**Describe the solution you'd like**
If the rules page would allow me to add my own command like e.g. that would do it. If it could be detected even better:
brave-browser --app="https://www.microsoft.com/en-us/microsoft-teams/join-a-meeting" --class=WebApp-TeamsMicrosoft6971 --name=WebApp-TeamsMicrosoft6971 --user-data-dir=/home/dom/.local/share/ice/profiles/TeamsMicrosoft6971

**Describe alternatives you've considered**
I have not found any alternative way to implement this

**Additional context**
N/A

**Comment by leancode** [comment]
Yes, that makes it show up and is a working workaround. Most certainly a step in the right direction. Great!

**Comment by ZeronoFreya** [comment]
Agreed, Windows also needs it.

**Comment by Overbryd** [comment]
Hi,

I have another use case for this on macOS, as I am running browsers in a virtual machine.

I would like Browsers to open a command line utility.
Therefore I would welcome being able to set an arbitrary command to be opened for an url.
I created myself an Apple Script which opens links in the browser inside the guest VM.

The following script can be called `$ remotebrowser https://google.com` and it will open https://google.com inside the default browser of the VM:

```osascript
#!/usr/bin/osascript

on run argv
  set remoteMachine to "eppc://vm1.local"
  tell application "Finder" of machine remoteMachine to open location (item 1 of argv)
end run
```

If Browsers could support opening links to the `remotebrowser` executable in my PATH, this would really make this setup very convenient.
